PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/survey/CustomerSurveyManager.java

```diff
@@ -114,10 +114,12 @@ private void updateSurveyStatus(ICustomerSurvey survey, CustomerSurveyResponse r
         if (response == CustomerSurveyResponse.NEVER_SHOW_AGAIN) {
             customerSurveyConfiguration.setAcceptSurvey(false);
         } else {
-            final int delay = response == CustomerSurveyResponse.ACCEPT ? TAKE_SURVEY_DELAY_BY_DAY : PUT_OFF_DELAY_BY_DAY;
             final CustomerSurveyConfiguration.CustomerSurveyStatus surveyStatus = getSurveyStatus(survey);
-            surveyStatus.setSurveyTimes(surveyStatus.getSurveyTimes() + 1);
-            surveyStatus.setLastSurveyDate(LocalDateTime.now());
+            if (response == CustomerSurveyResponse.ACCEPT) {
```

Comment:
we should have discussed the design about this feature. I have some different opinions about current implementation.